platform/qt/src/qmapboxgl.cpp

```diff
@@ -562,21 +621,21 @@ Coordinate QMapboxGL::coordinate() const
     return Coordinate(latLng.latitude, latLng.longitude);
 }
 
-void QMapboxGL::setCoordinate(const Coordinate &coordinate_)
+void QMapboxGL::setCoordinate(const QMapbox::Coordinate &coordinate_)
 {
     d_ptr->mapObj->setLatLng(mbgl::LatLng { coordinate_.first, coordinate_.second }, d_ptr->margins);
 }
 
 /*!
-    Convenience method for setting the \a coordinate and \a zoom simultaneously.
+    Convenience method for setting the \a coordinate_ and \a zoom_ simultaneously.
```

Comment:
That is true. I've been trying when possible and being creative with the names. 

In this case, the Qt standard is to use `coordinate()` as a getter instead of `getCoordinate()` and this often overlaps with parameters names. For Qt itself it is not a problem because it doesn't care about parameter shadowing.

One solution would be do a post processing of the generated documentation or disable this warning for the Qt build.